cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,15 @@
+describe(`Login Test`, () => {
+  it(`should log in with valid credentials`, () => {
+    cy.visit(`https://maglus02.github.io/social-media-client/`);
```

Comment:
```suggestion
    cy.visit(`/`);
```

You are not targeting the code in your current branch, but instead the finished code that has been merged into main. This means that if you create a new feature and a test for this new feature - the test would fail as the new feature is not present on the deployed site.